youtube-dl/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM wernight/youtube-dl
```

Comment:
Why did the base image change?

Reply:
The old one was 200mb and running a version from 2014, which was breaking on 'new' youtube videos.